cloud-gateway-service/src/main/java/org/zowe/apiml/cloudgatewayservice/config/ConnectionsConfig.java

```diff
@@ -195,12 +210,56 @@ public CloudEurekaClient primaryEurekaClient(ApplicationInfoManager manager, Eur
         AbstractDiscoveryClientOptionalArgs<?> args = new MutableDiscoveryClientOptionalArgs();
         args.setEurekaJerseyClient(eurekaJerseyClient);
 
-        CloudEurekaClient cloudEurekaClient = new CloudEurekaClient(appManager, config, args,
-            this.context);
+        final CloudEurekaClient cloudEurekaClient = new CloudEurekaClient(appManager, config, args, this.context);
         cloudEurekaClient.registerHealthCheck(healthCheckHandler);
         return cloudEurekaClient;
     }
 
+    @Bean
+    public List<AdditionalRegistration> additionalRegistration(StandardEnvironment environment) {
+        List<AdditionalRegistration> additionalRegistrations = new AdditionalRegistrationParser().extractAdditionalRegistrations(System.getenv());
+        log.debug("Parsed {} additional registrations, \t first: {}", additionalRegistrations.size(), additionalRegistrations.stream().findFirst().orElse(null));
```

Comment:
Why is logged just the first one? I guess it makes sense rather to log all of them, especially since it is just the debug level...